golang/cosmos/x/swingset/keeper/keeper.go

```diff
@@ -32,10 +33,22 @@ const (
 	StoragePathBundles      = "bundles"
 )
 
-const MaxUint53 = 9007199254740991 // Number.MAX_SAFE_INTEGER = 2**53 - 1
+const maxUint256Str = "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
```

Comment:
Please avoid easy-to-mistype constants.
```suggestion
var maxUint256Str = "0x" + strings.Repeat("f", 256/4)
```

Reply:
Looking for the best way to create the `2 ** 256 - 1` value. I thought embedding it as a constant would be the most obvious without needing BigInt operations. I feel we're moving away from that here. Open to alternatives!

Reply:
Hmm.  I'm not sure about this, since IIUC, there's code below that goes to great lengths to ensure the queue length cannot be more than 32 bits.

I don't think there's any reason to stay away from bitwise or bigint operations, especially if they're more straightforward than counting characters in a string and trying to reverse-engineer the math in order to verify correctness.

Reply:
At one point I was considering using the golang bigint on which sdk.Int is based off to compute the value. Would that be better ?